### DISCUSSION TITLE
Is it possible to load a bunch of file previously saved ?

### DISCUSSION THREAD

**Discussion by AdemIssadWytopia** [yasirkula/UnityNativeGallery#315] [Q&A]
Hello,
I want in my app, to allow users to save file into the galery (image, videos and sound), into a  specific folder.
When the user reconnect, I would like to load those file during splashscreen, meaning :
I would like to read the folder, get all files, and load them programmatically.

How to do that ? automaticcaly load all files in user storage folder ?

**Comment by yasirkula** [comment]
SaveImageToGallery's callback function returns a filepath. On Android, sometimes (but not always) it points to the path of the saved image. On iOS, it'll never point to the saved image's path. So NativeGallery doesn't provide a reliable way to achieve this. You need to keep a separate copy of the image in persistentDataPath and load it from there.
